dracut/anaconda-error-reporting.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Print what went wrong during the boot from installation perspective because Dracut seems
+# to timeout soon!
+#
+
+warn "############# Anaconda installer errors begin #############"
+warn "#                                                         #"
+warn "Seems that the boot has failed. The probable reason could"
+warn "be missing inst.stage2 or inst.repo kernel boot parameters"
+warn "on the kernel cmdline. Please verify that you have"
+warn "inst.stage2 or inst.repo specified."
+warn "Please note that the 'inst.' prefix is now mandatory!"
```

Comment:
Some suggestions for the long text.

```suggestion
warn "It seems that the boot has failed. The probable reason could"
warn "be missing 'inst.stage2' or 'inst.repo' boot parameters"
warn "on the kernel cmdline. Please verify that you have"
warn "specified the inst.stage2 or inst.repo boot option."
warn "Please note that the 'inst.' prefix is now mandatory!"
```

Even that's not perfect, but I didn't want to write it all anew. Review by a native speaker could be a good idea.

Reply:
@snbueno if you have a time, could you please take a look how to improve the message? I suck in this :D

Reply:
TBH, so do I, usually I'm stuck in a "can't unread" state...

Reply:
Sorry this is not going to be formatted, but this is the error text I would suggest:

> The attempt to boot has failed. Possible causes include missing inst.stage2 or inst.repo boot parameters on the kernel cmdline. Please verify that you have specified inst.stage2 or inst.repo.
> Please also note that the 'inst.' prefix is now mandatory.

I replaced the "It seems..." language because people will only see this if booting has failed, and it is better to use declarative language.

I also just dislike exclamation marks. :joy: But obviously feel free to edit this as you wish. :smiley: 

What do you think?

Reply:
I'm not sure if the errors are actually always meaningful. It might be that Jirka wanted to express that uncertainty with the "it seems". But that's only a guess, he would have to confirm himself.

+1 to no exclamation marks.

Reply:
I like this text. And yes, I wanted to express "uncertainty" as you said and I think the `Possible causes...` is exactly that :).

Reply:
Updated. Thanks a lot for suggestions!